app/controllers/vehicle_usage_sets_controller.rb

```diff
@@ -187,8 +187,11 @@ def vehicle_usage_set_params
                                               :service_time_start,
                                               :service_time_end,
                                               :work_time,
-                                              :max_distance)
+                                              :max_distance,
+                                              :max_ride_distance,
+                                              :max_ride_duration)
     parameters[:max_distance] = DistanceUnits.distance_to_meters(parameters[:max_distance], @current_user.prefered_unit) if parameters.key?(:max_distance)
+    parameters[:max_ride_distance] = DistanceUnits.distance_to_meters(parameters[:max_ride_distance], @current_user.prefered_unit) if parameters.key?(:max_ride_distance)
```

Comment:
Missing max_ride_duration here?

Reply:
max_ride_duration is a duration, it does not require a conversion to the prefered_unit which only apply on distances